src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -27,6 +31,24 @@
     private static final String VALID_LOYALTY_POINTS = "1234";
     private static final int VALID_LOYALTY_POINTS_INT = 1234;
 
+    private static final String INVALID_INGREDIENTNAME_INTEGER = "10";
```

Comment:
Constant names must be all uppercase using underscore to separate words.
ie INVALID_INGREDIENT_NAME_INTEGER